src/modules/antislow.js

```diff
@@ -0,0 +1,25 @@
+/*
+    The class representing a normal speed spot.
+
+    Implements draw().
+*/
+class ReverseSlow {
```

Comment:
Maybe we can be consistent on the naming - I like ReverseSlow! Maybe we can rename this file to `reverseslow.js` or something similar instead of `antislow.js`? That way the file name matches the class name.